docs/UserGuide.md

```diff
@@ -180,15 +177,15 @@ Format: `edit <choose only 1> [i:INDEX] [s:STUDENT_ID] [e:EMAIL] [h:TELEGRAM_HAN
 
 Examples:
 *  `edit i:1 s/A0001234A e/johndoe@example.com` Edits the student id and email address of the 1st student to be `A0001234A` and `johndoe@example.com` respectively.
-  ![result for 'edit 1 s/A1654327X e/johndoe@example.com'](images/edit.png)
+  ![result for 'edit i:1 s/A1654327X e/johndoe@example.com'](images/edit.png)
 *  `edit s:A1234567X n/Betsy Crower t/` Edits the name of the student with a student ID of `A1234567X` to be `Betsy Crower` and clears all existing tags.
 *  `edit e:christian@nus.com h/@christan c/` Edits the telegram handle and clears the timetable of the student with an email of `christian@nus.com`.
 *  `edit h:@christan c/mon: 8-10, 10-12 tue: 11-13 thu: 12-15, 15-17` Replaces the timetable of the student with a telegram handle of `@christan` with this new one according to
    the specifications in the command
 
 ### Locating students by name: `find`
 
-Finds students whose criteria match any of the given keywords.
+Finds students whose criteria match all the given keywords.
```

Comment:
Should we include that the find criteria uses fuzzy matching for each of the given keyword? It uses fuzzy matching but the students must match all the given keywords.

Reply:
![image](https://github.com/AY2324S2-CS2103-T14-1/tp/assets/113418549/70f434ff-1244-4234-96b6-28f142633c07)
Ive updated this part while leaving `Finds students whose criteria match all the given keywords.` alone